docs/source/metrics/divisions.md

```diff
@@ -1,11 +1,58 @@
 # Division Metrics
 
-:::{note}
-Coming soon!
-:::
+Division metrics are computed using the annotations described in {doc}`../track_errors/divisions`.
 
 :::{warning}
 These metrics are written assuming that the ground truth annotations are dense. If that is not the case, interpret the numbers carefully. Consider eliminating metrics that use the number of false positives (including Precision and F1 Score).
 :::
 
+Division metrics can be calculated as follows:
+
+```python
+from traccuracy.matchers import PointMatcher
+from traccuracy.metrics import DivisionMetrics
+
+# Load data using a function of your choice from traccuracy.loaders
```

Comment:
As above, make mention of being able to construct the TrackingGraph without a loader